---
layout: page
title: Publications
permalink: /publications/
---
---

### PEER-REVIEWED PUBLICATIONS

1. Via A, Attwood TK, Fernandes PL, Morgan SL, Schneider MV, Rustici G, and Tractenberg RE. A new pan-European Train-the-Trainer Programme for bioinformatics: Pilot results on feasibility, utility and sustainability of learning. Submitted to Briefings in Bioinformatics.
2. Jiménez RC et al. (2017) Four simple recommendations to encourage best practices in research software. F1000Research. doi: http://dx.doi.org/10.12688/f1000research.11407.1
3.	Schiffthaler B, Kostadima M, the NGS Trainer Consortium, Delhomme N and Rustici G (2016) Training in High-Throughput Sequencing: Common Guidelines to Enable Material Sharing, Dissemination and Re-Usability. PLoS Computational Biology, 12(6): 
e1004937. doi: 10.1371/journal.pcbi.1004937 
4.	Ison J, Rapacki K, Ménager H, Kalaš M, Rydza E, Chmura P, Anthon C, Beard N, Berka K, Bolser D, Booth T, Bretaudeau A, Brezovsky J, Casadio R, Cesareni G, Coppens F, Cornell M, Cuccuru G, Davidsen K, Vedova GD, Dogan T, Doppelt-Azeroual O, Emery L, Gasteiger E, Gatter T, Goldberg T, Grosjean M, Grüning B, Helmer-Citterich M, Ienasescu H, Ioannidis V, Jespersen MC, Jimenez R, Juty N, Juvan P, Koch M, Laibe C, Li JW, Licata L, Mareuil F, Mičetić I, Friborg RM, Moretti S, Morris C, Möller S, Nenadic A, Peterson H, Profiti G, Rice P, Romano P, Roncaglia P, Saidi R, Schafferhans A, Schwämmle V, Smith C, Sperotto MM, Stockinger H, Vařeková RS, Tosatto SC, de la Torre V, Uva P, Via A, Yachdav G, Zambelli F, Vriend G, Rost B, Parkinson H, Løngreen P, Brunak S. (2016) Tools and data services registry: a community effort to document bioinformatics resources. Nucleic Acids Res. 44(D1):D38-47. doi: 10.1093/nar/gkv1116. 
5.	 Lapatas V, Stefanidakis M, Jimenez RC, Via A, Schneider MV (2015) Data integration in biological research: an overview. J Biol Res (Thessalon). 22(1):9. doi: 10.1186/s40709-015-0032-5 
6.	Attwood TK, Bongcam-Rudloff E, Brazas ME, Corpas M, Gaudet P, Lewitter F, Mulder N, Palagi PM, Schneider MV, van Gelder CW; GOBLET Consortium (2015) GOBLET: the Global Organisation for Bioinformatics Learning, Education and Training. PLoS Comput Biol. 11(4):e1004143. doi: 10.1371/journal.pcbi.1004281. 
7.	Via A, Zanzoni A (2015) A prismatic view of protein phosphorylation in health and disease. Front. Genet. doi: 10.3389/fgene.2015.00131 
8.	Via A, Uyar B, Brun C, Zanzoni A. (2015) How pathogens use linear motifs to perturb host cell networks. Trends Biochem Sci. 40(1):36-48. doi: 10.1016/j.tibs.2014.11.001. 
9.	Di Marino D, Coletta A, D'Annessa I, Via A*, Tramontano A. (2015) Characterization of the differences in the cyclopiazonic acid binding mode to mammalian and P. falciparum Ca2+ pumps: a computational study. PROTEINS: Structure, Function and Bioinformatics. Jan 10. doi: 10.1002/prot.24734. 
10.	Dias RO, Via A, Brandão MM, Tramontano A, Silva-Filho MC. (2015) Digestive peptidase evolution in holometabolous insects culminated in a specialized group of enzymes in Lepidoptera. Journal of Molecular Biology. 58C:1-11. doi: 10.1016/j.ibmb.2014.12.009. 
11.	Corpas M, Jimenez RC, Bongcam-Rudloff E, Budd A, Brazas MD, Fernandes PL, Gaeta B, van Gelder C, Korpelainen E, Lewitter F, McGrath A, MacLean D, Palagi PM, Rother K, Taylor J, Via A, Watson M, Schneider MV, Attwood TK. (2015) The GOBLET training portal: a global repository of bioinformatics training materials, courses and trainers. Bioinformatics. 31(1):140-2. doi: 10.1093/bioinformatics/btu601. 
12.	Fiorillo A, Di Marino D, Bertuccini L, Via A, Pozio E., Ilari A, Lalle M. (2014) The crystal structure of Giardia duodenalis g14-3-3 in the apo form reveals unique features of the 14-3-3 protein family. PLoS ONE. 9(3): e92902. doi: 13.1002/prot.24734. 
12.	Via A, Blicher T, Bongcam-Rudloff E, Brazas MD, Brooksbank C, Budd A, De Las Rivas J, Dreyer J, Fernandes PL, van Gelder C, Jacob J, Jimenez RC, Loveland J, Moran F, Mulder N, Nyrönen T, Rother K, Schneider MV, Attwood TK. (2013) Best practices in bioinformatics training for life scientists. Briefings in Bioinformatics. 14(5):528-537. doi: 10.1093/bib/bbt043. 
14.	Jimenez RC, Albar JP, Bhak J, Blatter MC, Blicher T, Brazas MD, Brooksbank C, Budd A, De Las Rivas J, Dreyer J, van Driel MA, Dunn MJ, Fernandes PL, van Gelder CW, Hermjakob H, Ioannidis V, Judge DP, Kahlem P, Korpelainen E, Kraus HJ, Loveland J, Mayer C, McDowall J, Moran F, Mulder N, Nyronen T, Rother K, Salazar GA, Schneider R, Via A, Villaveces JM, Yu P, Schneider MV, Attwood TK, Corpas M. (2013) iAnn: an event sharing platform for the life sciences. Bioinformatics. 29(15):1919-1921. doi: 10.1093/bioinformatics/btt306. 
15.	Lepore R, Tramontano A, Via A (2013) TIPs: a database of therapeutic targets in pathogens and associated tools. Bioinformatics. 29(14):1821-1822. doi: 10.1093/bioinformatics/btt289. 
16.	Pillai DR, Lau R, Khairnar K, Lepore R, Via A, Staines H.M., Krishna S (2012) Artemether resistance in vitro is linked to mutations in PfATP6 in travellers returning with Plasmodium falciparum infections. Malar J. 11:131. doi: 10.1186/1475-2875-11-131. 
17.	Caroli A, Simeoni S, Lepore R, Tramontano A, Via A (2012) Investigation of a Potential Mechanism for the Inhibition of SmTGR by Auranofin and its implications for Plasmodium falciparum Inhibition. Biochem Biophys Res Commun. 417(1):576-581. doi: 10.1016/j.bbrc.2011.12.009; 
18.	Dinkel H, Michael S, Weatheritt RJ, Davey NE, Van Roey K, Altenberg B, Toedt G, Uyar B, Seiler M, Budd A, Jödicke L, Dammert MA, Schroeter C, Hammer M, Schmidt T, Jehl P, McGuigan C, Dymecka M, Chica C, Luck K, Via A, Chatr-Aryamontri A, Haslam N, Grebnev G, Edwards RJ, Steinmetz MO, Meiselbach H, Diella F, Gibson TJ (2012) ELM--the database of eukaryotic linear motifs. Nucleic Acids Res. 40:D242-D251. doi: 10.1093/nar/gkr1064. 
19.	Schneider MV, Walter P, Blatter MC, Watson J, Brazas MD, Rother K, Budd A, Via A, van Gelder C, Jacob J, Fernandes P, Nyronwn T, De Las Rivas J, Blicher T, Loveland J, McDowall J, Jones P, Vaughan BW, Lopez R, Attwood TK, Brooksbank C (2012) Bioinformatics Training Network (BTN): A community resource for bioinformatics trainers. Brief Bioinform 13(3):383-389. doi: 10.1093/bib/bbq021. 
20.	Via A, De Las Rivas J, Attwood TK, Landsman D, Brazas MD, Leunissen JAM, Tramontano A, Schneider MV (2011) 10 simple rules for developing a short bioinformatics training course. PLoS Computational Biology. 7(10):e1002245. doi: 10.1371/journal.pcbi.1002245. 
21.	Sayadi A, Briganti L, Tramontano A, Via A (2011) Exploiting Publicly Available Biological and Biochemical Information for the Discovery of Novel Short Linear Motifs. PLoS One 6(7): e22270. doi: 10.1371/journal.pone.0022270. 
21.	Lepore A, Simeoni S, Raimondo D, Caroli A, Tramontano A, Via A (2011) Identification of the Schistosoma mansoni molecular target for the anti malarial drug artemether. J Chem Inf Model 51(11):3005-3016. doi: 10.1021/ci2001764. 
23.	Mancini E, Tammaro F, Baldini F, Via A, Raimondo D, George P, Audisio P, Sharakhov I, Tramontano A, Catteruccia F, della Torre A (2011) Molecular evolution of a gene cluster of female serine proteases involved in reproductive processes of the malaria mosquito Anopheles gambiae. BMC Evolutionary Biology 11: 72. doi: 10.1186/1471-2148-11-72. 
24.	Dinkel H, Chica C, Via A, Gould C, Jensen L, Gibson T, Diella F (2011) Phospho.ELM: a database of phosphorylation sites- update 2011 Nucleic Acid Research, 39:D261-D267. doi: 10.1093/nar/gkq1104. 
25.	Via A, Diella F, Gibson TJ, Helmer-Citterich M (2011) From sequence to structural analysis in protein phosphorylation motifs. Frontiers in Bioscience 16: 1261-1275. doi: 10.2741/3787. 
26.	Zanzoni A, Carbajo D, Diella F, Gherardini PF, Tramontano A, Helmer-Citterich M, Via A (2011) Phospho3D 2.0: An enhanced database of three-dimensional structures of phosphorylation sites. Nucleic Acid Research 39: D268-D271. doi: 10.1093/nar/gkl922; 
27.	Chinappi M†, Via A†, Marcatili P, Tramontano A (2010) On the mechanism of chloroquine resistance in Plasmodium falciparum. PLoS ONE 5(11):e14064. Doi: 10.1371/journal.pone.0014064. 
28.	Schneider MV, Watson J, Attwood T, Rother K, Budd A, McDowall J, Via A, Fernandes P, Nyronen T, Blicher T, Jones P, Blatter M-C, De Las Rivas J, Judge DP, van der Gool W, Brooksbank C (2010) Bioinformatics training: a review of challenges, actions and support requirements. Briefings in Bioinformatics 11(6): 544-551. doi: 544-551. 10.1093/bib/bbq021. 
29.	Gould CM, Diella F, Via A, Puntervoll P, Gemünd C, Chabanis-Davidson S, Michael S, Sayadi A, Bryne JC, Chica C,  Seiler M, Davey NE, Haslam N, Weatheritt RJ, Budd A, Hughes T, Pas J, Rychlewski L, Travé G, Aasland R, Helmer-Citterich M, Linding R, Gibson TJ. (2010) ELM: The status of the 2010 Eukaryotic Linear Motif Resource. Nucleic Acid Research 38:D167-D180. Doi: 10.1093/nar/gkp1016. 
30.	Via A, Gould CM, Gemünd C, Gibson TJ, Helmer-Citterich M (2009) A structure filter for the Eukaryotic Linear Motif Resource. BMC Bioinformatics 10:351. doi: 10.1186/1471-2105-10-351. 
31.	Diella F, Gould C, Chica C, Via A, Gibson TJ (2008) Phospho.ELM: a database of phosphorylation sites - update 2008. Nucleic Acid Res 36:D240-D244. doi: 10.1093/nar/gkm772. 
32.	Ausiello G, Gherardini PF, Marcatili P, Tramontano A, Via A, Helmer-Citterich M (2008) FunClust: a web server for the identification of structural motifs in a set non-homologous protein structures. BMC Bioinformatics 9:S2. doi: 10.1186/1471-2105-9-S2-S2. 
33.	Ferraro E, Peluso D, Via A, Ausiello G, Helmer-Citterich M (2007) SH3-Hunter: discovery of SH3 domain interaction sites in proteins. Nucleic Acid Res 35:W451-W454. doi: 10.1093/nar/gkm296. 
34.	Via A, Peluso D, Gherardini PF, de Rinaldis E, Colombo T, Ausiello G, Helmer-Citterich M (2007) 3dLOGO: a web server for the identification, analysis and use of conserved protein substructures. Nucleic Acid Res 35:W416-W419. doi: 10.1093/nar/gkm228. 
35.	Via A, Gherardini F, Ferraro E, Ausiello G, Scalia Tomba G, Helmer-Citterich M (2007) False occurrences of functional motifs in protein sequences highlight evolutionary constraints. BMC Bioinformatics 8(1):68. doi: 10.1186/1471-2105-8-68. 
36.	Ausiello G, Peluso D, Via A, Helmer-Citterich M (2007). Local comparison of protein structures highlights cases of convergent evolution in analogous functional sites. BMC Bioinformatics 8:S24. doi: 10.1186/1471-2105-8-S1-S24. 
37.	Zanzoni A, Ausiello G, Via A, Gherardini PF, Helmer-Citterich M (2006) Phospho3D: a database of three-dimensional structures of protein phosphorylation sites. Nucleic Acid Res 35:D229-D231. doi: 10.1093/nar/gkl922. 
38.	Ferraro E, Via A, Ausiello G, Helmer-Citterich M (2006) A novel structure-based encoding for machine-learning applied to the prediction of SH3 domain specificity. Bioinformatics 22(19):2333-2339. doi: 10.1093/bioinformatics/btl403. 
39.	Ferraro E, Via A, Ausiello G, Helmer-Citterich M (2005) A neural strategy for the inference of SH3 domain-peptide interaction specificity. BMC Bioinformatics 6 Suppl 4: S13. doi: 10.1186/1471-2105-6-S4-S13. 
40.	Ausiello G, Via A, Helmer-Citterich M (2005) Query3d: a new method for high- throughput analysis of functional residues in protein structures. BMC Bioinformatics 6 Suppl 4: S5. doi: 10.1186/1471-2105-6-S4-S5. 
41.	Ausiello G, Zanzoni A, Peluso D, Via A, Helmer-Citterich M (2005) pdbFun: Mass selection and fast comparison of annotated PDB residues. Nucleic Acids Res 33: W133-W137. doi: 10.1093/nar/gki499; 
42.	Via A, Zanzoni A, Helmer-Citterich M. (2005) Seq2Struct: a resource for establishing sequence-structure links Bioinformatics 21(4): 551-553. doi: 10.1093/bioinformatics/bti049 
43.	Diella F, Cameron S, Gemund C, Linding R, Via A, Kuster B, Sicheritz-Ponten T, Blom N, Gibson TJ (2004) Phospho.ELM: a database of experimentally verified phosphorylation sites in eukaryotic proteins. BMC Bioinformatics 5(1):79. 
44.	Via A, Helmer-Citterich M (2004) A structural study for the optimisation of functional motifs encoded in protein sequences. BMC Bioinformatics 5(1):50. doi: 10.1186/1471-2105-10-351. 
45.	Ferrè F, Via A, Ausiello G, Brannetti B, Zanzoni A, Helmer-Citterich M (2003) Development of computational tools for the inference of protein interaction specificity rules and functional annotation using structural information. Comparative and Functional Genomics 4(4); 416-419. doi: 10.1002/cfg.304. 
46.	Puntervoll P, Linding R, Gemund C, Chabanis-Davidson S, Ramu C, Mattingsdal M, Cameron S, Martin DMA, Ausiello G, Brannetti B, Costantini A, Ferrè F, Maselli V, Via A, Cesareni G, Diella F, Superti-Furga G, Wyrwicz L, McGuigan C, Gudavalli R, Letunic I, Bork P, Rychlewski L, Küster B, Helmer-Citterich M, Hunter WN, Aasland R, Gibson TJ (2003) The ELM server: A new source for revealing short functional sites in modular eukaryotic proteins. Nucleic Acids Res 31(13):3625-3630. doi: 10.1093/nar/gkg545. 
47.	Via A, Ferrè F, Brannetti B, Helmer Citterich M (2000) Protein surface similarities: a survey of methods to describe and compare protein surfaces. Cell Mol Life Sci., 57 (13-14):1970-1977. doi: 10.1007/PL00000677. 
48.	Via A, Ferrè F, Brannetti B, Valencia A, Helmer Citterich M (2000) Three-dimensional view of the surface motif associated to the ploop structure: cis and trans cases of convergent evolution. J. Mol. Biol 303 (4): 455-465. doi:10.1006/jmbi.2000.4151 
49.	Brannetti B, Via A, Cesareni G, Helmer Citterich M (2000) SH3-SPOT: an algorithm to predict preferred ligands to different members of sh3 gene family. J. Mol. Biol 298(2):313-328. doi: 10.1006/jmbi.2000.3670. 


### BOOKS AND BOOK CHAPTERS
- Via A, Rother K, Tramontano A (2014) Managing your Biological Data with Python. Series: Chapman & Hall/CRC Mathematical and Computational Biology; Chapman and Hall/CRC Press. Taylor & Francis Group. ISBN: 9781439880937.

- Via A, Tramontano A (2011) Protein structural motifs: identification, annotation and use in function prediction. In Sequence and Genome Analysis II – Methods and Applications. ISBN: 978-1463789138. iConcept Press (http://www.iconceptpress.com/books/sequence-and-genome-analysis-ii--methods-and-applications/) 
- Brannetti B†, Via A† (2002) Theoretical aspects of Protein Sequence Alignments. In The Internet for Cell and Molecular Biologists - ed. A. Cabibbo, R.P. Grant, M. Helmer-Citterich, Horizon Scientific Press, pg. 73-87.

- Via A (2002) Practical aspects of Protein Sequence Analysis. In The Internet for Cell and Molecular Biologists - ed. A. Cabibbo, R.P. Grant, M. Helmer-Citterich, Horizon Scientific Press, pg. 163-232.
